pygments/lexers/mumps.py

```diff
@@ -0,0 +1,915 @@
+"""
+    pygments.lexers.mumps
+    ~~~~~~~~~~~~~~~~~~~~~
+
+    Lexers for MUMPS
+
+    :copyright: Copyright 2006-2022 by the Pygments team, see AUTHORS.
+    :license: BSD, see LICENSE for details.
+"""
+
+import re
+from pygments.lexer import ExtendedRegexLexer, words, bygroups, default, include
+from pygments.token import Whitespace, Keyword, Operator, Number, Name, Punctuation, Comment, String
+
+__all__ = ['MumpsLexer']
+
+class MumpsLexer(ExtendedRegexLexer):
+    """
+    For MUMPS source code.
+
+    Derived from `The Annotated M[UMPS] Standard <http://71.174.62.16/Demo/AnnoStd>`
+    Section numbers below refer to the sections from the 1995 (current) standard on that site
```

Comment:
This line should be a comment since the docstring ends up in the documentation (https://pygments.org/docs/lexers/) where you don't see the code.

Reply:
This is down to a single line now.